src/node/cli.ts

```diff
@@ -212,7 +179,6 @@ const options: Options<Required<Args>> = {
     description: "Force to open a file or folder in an already opened window.",
   },
 
-  locale: { type: "string" },
```

Comment:
Do we know if anyone uses this flag and removing would cause a regression?  I think this is something to do with display language support but am not sure it is useful.

Reply:
I am also not sure if it's used. I guess we'll have to make this a breaking change and if there is a regression, we'll fix after?

Reply:
(I wish there were an easier way to know if this is actually used)

Reply:
AFAIK locale isn’t supported in the server arguments, and they seem to be migrating to a client driven approach, [here](https://github.com/Microsoft/vscode/blob/8b8e1d1dfaf6fa6e42c4ca2743c40a012c4ac455/src/vs/server/extensionHostConnection.ts#L26) and [here](https://github.com/microsoft/vscode/blob/8b8e1d1dfaf6fa6e42c4ca2743c40a012c4ac455/src/vs/server/remoteAgentEnvironmentImpl.ts#L124)